netebpfext/net_ebpf_ext_tracelog.h

```diff
@@ -105,6 +94,24 @@ net_ebpf_ext_trace_terminate();
         TraceLoggingKeyword((keyword)),                         \
         TraceLoggingString(message, "Message"));
 
+#define NET_EBPF_EXT_LOG_MESSAGE_STRING(trace_level, keyword, message, value) \
```

Comment:
These macros replace the prior definition using __VAR_ARGS__ as that worked only for 'string' arguments for the variable part and gave compilation errors for other types.